### ISSUE TITLE
Interpretation of allele codes for multiallelic variants that have been split into separate lines

### ISSUE THREAD

**Issue by amholleman** [samtools/bcftools#1764]
Hi,

Just a quick question/clarification:

Let's say a multiallelic variant has two ALT alleles: ALT1 and ALT2. _Before_ splitting this multiallelic variant, allele code '0' refers to the REF allele only. _After_ splitting this multiallelic variant using 'bcftools norm -m -', the single record for this variant in the VCF has been split into two records, one for ALT1 and one for ALT2. In these two post-split records, allele code '0' now refers to any allele other than the single ALT allele defined in the newly split record, and doesn't simply mean REF allele, correct? In other words, in the post-split record for ALT1, allele code '1' refers to the ALT1 allele and allele code '0' collectively refers to REF or ALT2 (i.e., anything other than ALT1)? I just want to be sure I understand this correctly.

Thanks very much!
Aaron

**Comment by amholleman** [comment]
Thanks!

**Comment by benostendorf** [comment]
Would it be worth considering to add the option to set non-REF and non-ALT1 alleles to NA/. instead of REF in this situation?
In my opinion there are situtations where you'd like to compare ALT1 to REF rather than ALT1 to all non-ALT1.

**Comment by pd3** [comment]
@benostendorf Do you mean literally set the GT field to be `NA/.`? Best if you provided a specific test case demonstrating the behavior you'd like

**Comment by pd3** [comment]
> Would it be worth considering to add the option to set non-REF and non-ALT1 alleles to NA/. instead of REF in this situation? In my opinion there are situtations where you'd like to compare ALT1 to REF rather than ALT1 to all non-ALT1.

Just added a new option --multi-overlap which is analogous to --atom-overlap and allows to select between the ref (0) or missing (.) allele (https://github.com/samtools/bcftools/commit/658a0993d43121ee1fbdfdcc29e3a849b1cf440f)

**Comment by gernophil** [comment]
Am I correct that this option is not integrated in the latest release (1.16) yet? If so, when will this happen?

**Comment by pd3** [comment]
@gernophil That's correct, you need the latest github version for that, see here how to install http://samtools.github.io/bcftools/howtos/install.html. I can't give a time estimate atm.
